section-06-model-serving-api/house-prices-api/requirements.txt

```diff
@@ -0,0 +1,9 @@
+uvicorn>=0.11.3,<0.12.0
+fastapi>=0.64.0,<1.0.0
+python-multipart>=0.0.5,<0.1.0
+pydantic>=1.8.1,<1.9.0
+typing_extensions>=3.7.4,<3.8.0
```

Comment:
are you discussing what these, probably new for many students, packages are? pydantic, python-multipart, uvicorn?
In the email you mentioned you introduce loguru, correct?